tests/test_environ.py

```diff
@@ -0,0 +1,4 @@
+from os import environ
+
+var = environ.get("PLOOMBER_VERSION_CHECK_DISABLED")
```

Comment:
this will not evaluate what we want. you need to add a new notebook to the documentation and such notebook should contain this code: https://ploomber-contributing.readthedocs.io/en/latest/documentation/notebooks.html

Reply:
I added the notebook (.md file) to `doc/user-guide/testing/vcheck.md` and running the following cell outputs None:

```sh
from os import environ
var = environ.get("PLOOMBER_VERSION_CHECK_DISABLED")
print(var)
```

Reply:
we need to run this in readthedocs (the service that builds our documentation), not locally.

so push the notebook vcheck.md to this PR, then we let readthedocs run and check if it gets the environment variable

the test will output true because that's how the unit tests are configured. but unit tests != documentation